### PR TITLE
feat: added image selection gallery

### PR DESCRIPTION
**Pull request by kylecalbert**

### CODE REVIEW

File: src/pages/Gallery.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import product1image from '../images/image-product-1.jpg';
+import product2image from '../images/image-product-2.jpg';
+import product3image from '../images/image-product-3.jpg';
+import product4image from '../images/image-product-4.jpg';
+import { colors } from '../common/constants/constants';
+export const ImageGalleryContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 1rem;
+  /* background-color: green; */
```
Comment (sukhvir-notta) [inline src/pages/Gallery.js]: Clean up comments if not needed

File: src/pages/Product.styled.js [hunk 2]
```diff
@@ -10,7 +10,12 @@ export const Container = styled.div`
 `;
 
 export const LeftContainer = styled.div`
-  background-color: purple;
+  display: flex;
+
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  /* background-color: purple; */
```
Comment (sukhvir-notta) [inline src/pages/Product.styled.js:20]: Clean up comments if not needed

File: src/pages/Gallery.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+import product1imagethumb from '../images/image-product-1-thumbnail.jpg';
+import product2imagethumb from '../images/image-product-2-thumbnail.jpg';
+import product3imagethumb from '../images/image-product-3-thumbnail.jpg';
+import product4imagethumb from '../images/image-product-4-thumbnail.jpg';
+import product1image from '../images/image-product-1.jpg';
+import product2image from '../images/image-product-2.jpg';
+import product3image from '../images/image-product-3.jpg';
+import product4image from '../images/image-product-4.jpg';
+import { colors } from '../common/constants/constants';
+export const ImageGalleryContainer = styled.div`
```
Comment (sukhvir-notta) [inline src/pages/Gallery.js]: Styling should be in a separate file to keep consistent with other components

File: src/components/ProductGallery.js
Comment (hamzahkhanAND) [inline src/components/ProductGallery.js:1]: Empty file?

File: src/pages/Gallery.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    img {
+      border: 0.2rem solid ${colors.primaryOrange};
+      border-radius: 1rem;
+    }
+  }
+`;
+
+export const Gallery = () => {
+  const [selectedImage, setSelectedImage] = useState(null);
```
Comment (sukhvir-notta) [inline src/pages/Gallery.js]: The default state for `selectedImage` should be the image for product 1 so it shows above the grid items when the page loads

File: src/pages/Gallery.js
Comment (sukhvir-notta) [inline src/pages/Gallery.js:1]: This can be moved to the components folder as it's being called in the Product page

**Pull request merged**